### ISSUE TITLE
Question and parameters passed

### ISSUE THREAD

**Issue by mattganis** [hanyazou/TelloPy#60]
I don't see it documented anywhere - but when I used the clockwise(x) method (or any other method that has the drone rotate) - what does the 'x' represent ?   

For the forward(x) method, I see the 'x' represents cm/sec speed - but I can't figure out the rotations

thanks.....Matt

**Comment by sonaurea** [comment]
[If you go to _internal folder then tello.py](
https://github.com/hanyazou/TelloPy/blob/develop-0.7.0/tellopy/_internal/tello.py)
```
    def clockwise(self, val):
        """
        Clockwise tells the drone to rotate in a clockwise direction.
        Pass in an int from 0-100.
        """
        log.info('clockwise(val=%d)' % val)
        self.left_x = val / 100.0
```
self.left_x is used in __send_stick_command() __recv_thread

If you want something more discrete try the [official tello official](https://dl-cdn.ryzerobotics.com/downloads/Tello/Tello%20SDK%202.0%20User%20Guide.pdf) commands or [easytello](https://github.com/Virodroid/easyTello/blob/master/easytello/tello.py) which uses rotation 0-360 degrees

perhaps something like this if using tellopy
```
command = "cw 360"
try:
    self.sock.sendto(command.encode('utf-8'), self.tello_addr)
```

**Comment by mattganis** [comment]
So looking at the code above:  self.left_x = val / 100.0

doe the value passed (val) represent a percent around a circle ?   For example if I pass 50, self.left_x is going to be 0.5 (so is that halfway around ?).

I guess I'm trying to figure out to tell it simply:  Rotate 180 degrees

**Comment by sonaurea** [comment]
No, I believe it is the speed, you can see it being used in the joystick example in the examples folder. So 100 is max speed turning clockwise, 0  is no speed turning clock wise.

From the official sdk,
You can just do this for a 0-360 discrete turn

```
command = "cw 180"
try:
    self.sock.sendto(command.encode('utf-8'), self.tello_addr)
```
https://dl-cdn.ryzerobotics.com/downloads/Tello/Tello%20SDK%202.0%20User%20Guide.pdf
![image](https://user-images.githubusercontent.com/15111182/75686792-cf27c400-5c6a-11ea-95ee-8b38ab2e9733.png)
